pkg/doc/verifiable/presentation.go

```diff
@@ -226,9 +232,58 @@ type rawPresentation struct {
 	RefreshService *RefreshService `json:"refreshService,omitempty"`
 }
 
+func (raw *rawPresentation) marshalJSON() ([]byte, error) {
```

Comment:
I think this function is redundant, same signature for both  `json.Marshal(raw) is equal to marshalJSON() `

Reply:
Yes, I think it makes sense. Probably there was smth extra in this function, but now it's just a wrapping over `json.Marshall()`.